### PR TITLE
Refactor cukinia configuration and update documentation

### PR DESCRIPTION
**Pull request by dupremathieu**

### CODE REVIEW

**Review by eroussy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# cukinia-tests
\ No newline at end of file
+# cukinia-tests
+
+This repository is a part of the [SEAPATH](https://www.lfenergy.org/projects/seapath/) project.
+It contained Cukinia's tests, which can be run on SEAPATH machines.
```
Comment (deribaucourt) [inline README.md]: It contains

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+These tests can be run and deployed using SEAPATH ansible playbook:
+* [test_deploy_cukinia.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_deploy_cukinia.yaml "test_deploy_cukinia.yaml") to install Cukinia on a machine
+* [test_deploy_cukinia_tests.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_deploy_cukinia_tests.yaml "test_deploy_cukinia_tests.yaml") to deploy tests
+* [test_run_cukinia.yaml](https://github.com/seapath/ansible/blob/debian-main/playbooks/test_run_cukinia.yaml "test_run_cukinia.yaml") to run the tests
+
+Test are installed in /etc/cukinia. It can be run manually with the cukinia command:
```
Comment (deribaucourt) [inline README.md]:
```suggestion
Tests are installed in /etc/cukinia. They can be run manually with the cukinia command:
```

File: README.md (hunk 1 above)
Comment (deribaucourt) [reply README.md]:
```suggestion
It contains Cukinia's tests, which can be run on SEAPATH machines.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
-# cukinia-tests
\ No newline at end of file
+# cukinia-tests
+
+This repository is a part of the [SEAPATH](https://www.lfenergy.org/projects/seapath/) project.
+It contained Cukinia's tests, which can be run on SEAPATH machines.
+These tests can be run and deployed using SEAPATH ansible playbook:
```
Comment (deribaucourt) [inline README.md]: The SEAPATH ansible

Comment (deribaucourt) [reply README.md]:
```suggestion
These tests can be run and deployed using the SEAPATH ansible playbook:
```

**Review by deribaucourt — COMMENTED** [review]
In commit message: 
 - "To avoid redundant files"

**Pull request merged**